R/non-cases.R

```diff
@@ -35,36 +39,56 @@ non_cases <- function() {
     "nc_pcos_1", 10, "20210101", "A10BA02", 5, "0000276",
     "nc_pcos_2", 10, "20190101", "A10BA02", 5, "0000276",
     "nc_pcos_3", 10, "20190101", "A10BA02", 5, "0000276",
+    "nc_preg_1", 10, "20180101", "A10", 5, "0000000",
+    "nc_preg_2", 10, "20180101", "A10", 5, "0000000",
+    "nc_preg_3", 10, "20200101", "A10", 5, "0000000",
+    "nc_preg_4", 10, "20200101", "A10", 5, "0000000",
   ) |>
     dplyr::bind_rows(sim_data$lmdb)
 
+  # LPR2 is before 2019
   lpr_adm <- tibble::tribble(
     ~pnr, ~c_spec, ~recnum, ~d_inddto,
     "nc_pcos_1", "08", "1", "20180101",
     "nc_pcos_2", "08", "1", "20170101",
     "nc_pcos_3", "08", "1", "20170101",
+    "nc_preg_1", "08", "1", "20180101",
+    "nc_preg_2", "08", "1", "20180101",
+    "nc_preg_1", "08", "2", "20180101",
+    "nc_preg_2", "08", "3", "20180101",
```

Comment:
The recnum should be unique across all rows to match the real data (not sure how to make these strings of only digits in the case data and stay sane)
```suggestion
    "nc_pcos_1", "08", "nc_pcos_1_rec_1", "20180101",
    "nc_pcos_2", "08", "nc_pcos_2_rec_1", "20170101",
    "nc_pcos_3", "08", "nc_pcos_3_rec_1", "20170101",
    "nc_preg_1", "08", "nc_preg_1_rec_1", "20180101",
    "nc_preg_2", "08", "nc_preg_2_rec_1", "20180101",
    "nc_preg_1", "08", "nc_preg_1_rec_2", "20180101",
    "nc_preg_2", "08", "nc_preg_2_rec_3", "20180101",
```